src/Metadata/Extractor/XmlExtractor.php

```diff
@@ -43,11 +43,23 @@ protected function extractPath(string $path)
                 'shortName' => $this->phpize($resource, 'shortName', 'string'),
                 'description' => $this->phpize($resource, 'description', 'string'),
                 'iri' => $this->phpize($resource, 'iri', 'string'),
-                'itemOperations' => $this->getAttributes($resource, 'itemOperation') ?: null,
```

Comment:
IMO you should create a dedicated method like this `XmlExtractor::getOperations($resource, $operationType)`.